src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -3619,6 +3619,7 @@ Activate by holding the key for the character you want to add an accent to, then
   </data>
   <data name="Alternate_OOBE_AlwaysOnTop_Title.Text" xml:space="preserve">
     <value>Always On Top</value>
+    <comment>{Locked="Always On Top"}</comment>
```

Comment:
We have a whole lot of these identical names! alas

Reply:
it is due to how xaml works.  we are locking on IDs w/ attributes